### ISSUE TITLE
qtranslate slug integration

### ISSUE THREAD

**Issue by The3IC** [herrvigg/qtranslate-x2#1]
Let me start of by saying that if this activity rekindles xTranslate, then that would be fantastic!  Unfortunately I am not a coder so cannot assist in the coding but thumbs up for any attempt to salvage qTranslate!

The issue: To enable that also URLs are translated/translatable, please consider merging qTranslate-slug (http://not-only-code.github.io/qtranslate-slug/). If I understood correctly, the slug plugin over-writes some parts of the original code as well? Not sure though.

**Comment by herrvigg** [comment]
Yes it is related but i'm just doing experiments in this project. The good one is https://github.com/herrvigg/qtranslate-x but i will move it to the new organization soon. The original project is quite in a messy state, it's completely abandoned and we have zero notice from its author for 2 years.

We'll try to revive it but it's quite a bit of work. My first goal is to relaunch it just for the minimal maintenance so that it can be used directly. For new features and other bug fixes it's another story.

I tried qtranslate-slug personally some time ago but i wasn't completely convinced. I don't remember exactly but i found some problems and i gave up. Also, Wordpress is quite a messy framework for integration (to not say horrible) so it's very fragile. It would require a lot of tests before full integration and release. So it might come in at a certain point but i don't think it will be very soon.
